tests/test_multisim_num_azim/results_true.dat

```diff
@@ -1,4 +1,4 @@
-# tracks: 116	# segments: 198
+# tracks: 116	# segments: 196
```

Comment:
Correct, 196 segments is the number of segments when no segments are split, which is the desired behavior.